### ISSUE TITLE
failing to parse ">/dev/null # a comment"

### ISSUE THREAD

**Issue by sp31415t1** [koalaman/shellcheck#2275]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC1073
- My shellcheck version (`shellcheck --version` or "online"): online
- [x] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [x] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [ ] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/bash

my_arr=( 'a' "b" )
echo "${my_arr[@]}" >/dev/null  # shellcheck - SC2034: my_arr appears unused.
echo "${my_arr[@]}"


```

#### Here's what shellcheck currently says:

By altering the comment or not using /dev/null, It's possible to pass... It looks like there is a reserved word (or something like that)
 
```
Line 4:
echo "${my_arr[@]}" >/dev/null  # shellcheck - SC2034: my_arr appears unused.
^-- SC1073: Couldn't parse this simple command. Fix to allow more checks.
                                ^-- SC1126: Place shellcheck directives before commands, not after.
>>                                                                           ^-- SC1072: Fix any mentioned problems and try again.
```

#### Here's what I wanted or expected to see:

no error

**Comment by sp31415t1** [comment]
Now, I read the second warning
https://github.com/koalaman/shellcheck/wiki/SC1126
I found my answer
>If this is not a directive and just a comment mentioning ShellCheck, please rewrite or capitalize
sorry for the noise
